src/sentry/tasks/post_process.py

```diff
@@ -760,6 +760,15 @@ def process_code_mappings(job: PostProcessJob) -> None:
             if event.data["platform"] not in SUPPORTED_LANGUAGES:
                 return
 
+            # Since we're currently only processing one issue per project per hour (see below),
+            # we want to get the best bang for our buck by making sure that we process a different
+            # issue each hour in a 24-hour period
+            issue_cache_key = f"code-mappings:group:{group_id}"
+            if cache.get(issue_cache_key) is None:
+                cache.set(issue_cache_key, True, 86400)  # 24 hours
+            else:
+                return
+
```

Comment:
~This should probably be after the "once per hour" check, otherwise you'll end up marking all groups as processed while you're ignoring this project~

Actually, this is kind of tricky. You really want to check that both `project_cache_key` is none, and `issue_cache_key` is none. If both are none, then set them both to true, otherwise do nothing.

Reply:
You're right - it needs to pass all the tests before it goes in the cache. Good catch, thanks!